db/migrate/20200806052718_create_profiles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :profiles do |t|
+      t.belongs_to :user, null: false, foreign_key: { on_delete: :cascade }
+      t.jsonb :data, null: false, default: {}
```

Comment:
Ensure we always deal with a JSON object. This is IMHO the most convenient way to deal with JSONB columns.

Reply:
Agree. It's much easier to remove nulls and leave a default :)